src/main/java/com/kaleido/fetch/service/ActivityService.java

```diff
@@ -83,40 +90,42 @@ public Activity getActivity(Long id) {
         return new Activity();
     }
     
-    public ResponseEntity<PlateMap> saveNewActivityDraft(PlateMap plateMap) {
+    public ResponseEntity<String> saveNewActivityDraft(PlateMap plateMap) {
     	
     	log.info("Platemap data is ", plateMap);
     	String plateMapURI = cabinetURI + "plate-maps";
-    	//ZonedDateTime currentTime = ZonedDateTime.now();
-    	//plateMap.setLastModified(currentTime);
-    	
     	RestTemplate restTemplate = new RestTemplate();
     	HttpHeaders headers = new HttpHeaders();
+    	String username = "admin";
+    	String password = "admin";
+    	headers.setBasicAuth(username, password);
```

Comment:
Is there a reason that we need authentication with Cabinet currently? I believe the save endpoint is unprotected?

Reply:
Sorry this was my local testing, as the remove authentication has some issue when testing locally, will remove it

Reply:
Done